### PR TITLE
KH-452: Re-organize to match the latest Ozone structure

### PR DESCRIPTION
**Pull request by rbuisson**
https://mekomsolutions.atlassian.net/browse/KH-452

Updated the start instructions as well to match the new structure. Those are documented in readme/impl-guide.md

FTR:
Build
```
./scripts/mvnw clean package [-Pprod]
```
Run
```
source [base|prod]/target/go-to-scripts-dir.sh
./start-demo.sh
```

### CODE REVIEW

**Review by ibacher — COMMENTED** [review]
Made a couple of notes. Some things I noticed:

1. This starts Postgres, even when nothing is using Postgres (that might be hard to do)
2. Not the biggest issue, but the RefApp uses a much newer version of MariaDB (10.8.2 vs 10.2)
3. I've got no clue why this is, but `destroy-demo.sh` doesn't seem to actually process the .env file.
4. (Very minor) it would be nice if the output URLs only included the services that were actually running.

File: scripts/go-to-scripts-dir.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+# Will be substituted at build time by the Maven Resource plugin filtering.
+cd ${project.build.directory}/${project.artifactId}-${project.version}/
+cd run/docker/scripts/
```
[the remark this answers is not in the document]
Comment (rbuisson) [reply scripts/go-to-scripts-dir.sh:5]: No particular reason for making it 2 lines.
I'll try out the `pushd` command which I never new about. Thanks

**Comment by rbuisson** [comment]
Thanks for the feedback @ibacher . I've modified the code accordingly.

**Review by kdaud — APPROVED** [review]
LGTM

**Pull request merged**